documentation/faq/60-env-vars.md

```diff
@@ -2,8 +2,6 @@
 title: How do I use environment variables?
 ---
 
-Vite uses [dotenv](https://github.com/motdotla/dotenv) to load environment variables from a file named `.env` or similar. Only environment variables prefixed with `VITE_` are exposed ([you can set `envPrefix` to change this](https://vitejs.dev/config/shared-options.html#envprefix)). You can access the variable using `import.meta.env.VITE_ENV_VAR`, and Vite will statically replace them at build-time.
+Environment variables are [loaded by Vite](https://vitejs.dev/guide/env-and-mode.html#env-files) from `.env` files and `process.env` during dev and build. These are available to your app via the [`$env/static/private`](/docs/modules#$env-static-private) and [`$env/static/public`](/docs/modules#$env-static-public) modules. Public variables — i.e. those that are safe to expose to users — must use a [prefix](/docs/configuration#kit-env-publicprefix) that defaults to `PUBLIC_`. Private variables cannot be imported into client-side code.
```

Comment:
I added this FAQ quite awhile ago when using Vite's environment variables was truly confusing. I think we should probably have some documentation of our own environment variable support in `documentation/docs/07-modules.md` and can remove this FAQ in favor of that

Reply:
we have https://kit-svelte-kxu7loqzq-svelte.vercel.app/docs/modules#$env-dynamic-private. did you mean something beyond that?

Reply:
No, that's good. I just didn't see `documentation/docs/07-modules.md` updated, but I guess that content gets inserted in when the docs are built

I'd say we just go ahead and remove this FAQ now then. No point in splitting documentation between multiple places

Reply:
cool, done — this means that searching for 'environment variables' will no longer favour the FAQ over the docs